localstack/services/kms/provider.py

```diff
@@ -133,182 +159,336 @@ def __init__(self, message=None):
         super().__init__("ValidationError", message=message)
 
 
-class KmsProvider(KmsApi):
-    @staticmethod
-    def get_kms_backend(context: RequestContext) -> BaseBackend:
-        return kms_backends[context.account_id][context.region]
+def _verify_key_exists(key_id, store: KmsStore):
+    if key_id not in store.keys:
+        raise NotFoundException(f"Invalid keyID '{key_id}'")
+
+
+def _get_key(any_type_of_key_id: str, store: KmsStore) -> KmsKey:
```

Comment:
nit: I'd probably make this part of the store itself, as a `get_key(..)` or `find_key(..)` method - seems an integral part of data/state access that the store itself could provide. (not critical, though..)

Reply:
Done.